layers/auto-completion/packages.el

```diff
@@ -269,7 +269,7 @@
         "Call `yas-expand' and switch to `insert state'"
         (interactive)
         (call-interactively 'aya-expand)
-        (evil-insert-state))
+        (unless holy-mode (evil-insert-state)))
```

Comment:
@abaw emac editing styles (aka holy-mode) is not the same thing as hybrid. You should be clear on what style you're referring to.